code/src/App.js

```diff
@@ -1,9 +1,54 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { FirstQuestion } from './components/FirstQuestion';
+import 'components-css/landing-page.css';
 
 export const App = () => {
+  const [visible, setVisible] = useState(false);
+  const [destinationOption, setDestinationOption] = useState();
+
+  const checkInput = () => {
+    setVisible(true);
+  };
```

Comment:
You could either change the name of this function to `startSurvey` or you could just change the state directly in the onclick. Eitherway - there's no input checking by clicking a button, right? ;)